src/cli.ts

```
@@ -2,13 +2,50 @@
 
 import { Command } from 'commander';
 import { version } from './utils/package-info.js';
+import { HeboAgent } from './agents/implementations/hebo-agent.js';
+import { ScoringService } from './scoring/scoring.service.js';
+import { EvaluationExecutor } from './evaluation/evaluation-executor.js';
+import { EvaluationConfig } from './evaluation/types/evaluation.types.js';
+import { Logger } from './utils/logger.js';
+import { EmbeddingProviderFactory } from './embeddings/config/embedding.config.js';
+import { readFileSync } from 'fs';
+import { join } from 'path';
```

ESLint

[error] 'join' is defined but never used.

(@typescript-eslint/no-unused-vars)